src/app/views/App.tsx

```diff
@@ -165,64 +172,86 @@ class App extends Component<IAppProps, IAppState> {
-            <div className={layout}>
-              {graphExplorerMode === Mode.Complete && <Authentication />}
-              {graphExplorerMode === Mode.TryIt && (
+        <FocusTrapZone>
+          <div className={`container-fluid ${classes.app}`}>
+            <div className='row'>
+              {graphExplorerMode === Mode.Complete && (
+                <div className={`col-sm-12 col-lg-3 col-md-4 ${classes.sidebar}`}>
+                  {showToggle && <IconButton
+                    iconProps={{ iconName: 'GlobalNavButton' }}
+                    className={classes.sidebarToggle}
+                    title='Remove sidebar'
+                    ariaLabel='Remove sidebar'
+                    onClick={() => this.toggleSidebar()}
+                  />}
+                  {showSidebar && <Sidebar />}
+                </div>
+              )}
+              <div className={layout}>
+                {graphExplorerMode === Mode.Complete && <Authentication />}
+                {graphExplorerMode === Mode.TryIt && (
+                  <div style={{ marginBottom: 8 }}>
+                    <MessageBar
+                      messageBarType={MessageBarType.warning}
+                      isMultiline={true}
+                    >
+                      <p>
+                        To try operations other than GET or to access your own data, sign in to
+                      <a className={classes.toGraphExplorer}
+                          tabIndex={0}
+                          href='https://developer.microsoft.com/en-us/graph/graph-explorer' target='_blank'>
+                          Graph Explorer.
+                      </a>
+                      </p>
+                    </MessageBar>
+                  </div>
+                )}
                 <div style={{ marginBottom: 8 }}>
+                  <QueryRunner onSelectVerb={this.handleSelectVerb} />
+                </div>
+                {error && (
                   <MessageBar
-                    messageBarType={MessageBarType.warning}
-                    isMultiline={true}
+                    messageBarType={MessageBarType.error}
+                    isMultiline={false}
+                    onDismiss={actions.clearQueryError}
                   >
-                    <p>
-                      To try operations other than GET or to access your own data, sign in to
-                      <a className={classes.toGraphExplorer}
-                        tabIndex={0}
-                        href='https://developer.microsoft.com/en-us/graph/graph-explorer' target='_blank'>
-                        Graph Explorer.
-                      </a>
-                    </p>
+                    {`${error.statusText} - ${error.status}`}
                   </MessageBar>
-                </div>
-              )}
-              <div style={{ marginBottom: 8 }}>
-                <QueryRunner onSelectVerb={this.handleSelectVerb} />
+                )}
+                {sampleQuery && (<div />)}
```

Comment:
I think it's safe to remove this line, I don't see `sampleQuery` passed as a prop if you look at `mapDispatchToProps ||  mapStateToProps` but there is `receivedSampleQuery` not sure if that's what should be there.

Reply:
@cwahome you can remove this now.